### ISSUE TITLE
Add a API to see answer question/requirement

### ISSUE THREAD

**Issue by MV-GH** [LemmyNet/lemmy#3461]
### Requirements

- [X] Is this a feature request? For questions or discussions use https://lemmy.ml/c/lemmy_support
- [X] Did you check to see if this issue already exists?
- [X] Is this only a feature request? Do not put multiple feature requests in one issue.
- [X] Is this a backend issue? Use the [lemmy-ui](https://github.com/LemmyNet/lemmy-ui) repo for UI / frontend issues.

### Is your proposal related to a problem?

I am trying to implement a in app registration form, in Jerboa. But for that I would need to know the requirements of a instance. I can already check if a instance requires a captcha by checking if get_captcha returns null. But currently there is no way to check if a instance requires a answer and the answer text above it.

Like this section
![image](https://github.com/LemmyNet/lemmy/assets/67873169/315528f8-f50b-4312-b8fd-8cb015b182ec)

### Describe the solution you'd like.

There should a be a similar API like get_captcha that returns the above text. If it is required or null if it is not required. 

### Describe alternatives you've considered.

A API that returns all the requirements of a instance, like `{answer: true, captcha: true, email_verification: true}`

**Comment by Nutomic** [comment]
You can get all this info from `/api/v3/site`.

**Comment by MV-GH** [comment]
Ah yes indeed, mb.


<details><summary> example </summary>

```json

"local_site": {
            "id": 1,
            "site_id": 1,
            "site_setup": true,
            "enable_downvotes": false,
            "enable_nsfw": true,
            "community_creation_admin_only": true,
            "require_email_verification": false,
            "application_question": "To combat spam, we have restricted user registration on this instance. Please write a short description answering ***all three of these things* or your account will be denied**:\n\n- What communities (our version of \"subreddits\") you would most like to participate in?\n- Why you would like to join Lemmy.one?\n- Why did you chose the username you did?\n\nAlso, just so you know before you register, \"community\" (subreddit equivalent) *creation* is [disabled](https://lemmy.one/post/41) on Lemmy.one, so if you were hoping to do that you might not want your account here.",
            "private_instance": false,
            "default_theme": "browser",
            "default_post_listing_type": "All",
            "legal_information": "## Legal Information\n\nLemmy.one is operated by [Fediverse Communications LLC](https://fediverse.us), a Minnesota (USA) company, and is hosted on servers operated in Germany and Finland. All content on this server is expected to be legal in all of these jurisdictions. If you have any copyright, trademark, legal concerns, please email us directly.\n\n- DMCA Complaints: dmca@fediverse.us\n- Trademark Complaints: trademark@fediverse.us\n- Abuse Complaints: abuse@fediverse.us\n\n## Privacy Policy\n\nThis privacy policy describes how Fediverse Communications LLC (\"lemmy.one\", \"we\", \"us\") collects, protects and uses the personally identifiable information you may provide through the lemmy.one website or its API. The policy also describes the choices available to you regarding our use of your personal information and how you can access and update this information. This policy does not apply to the practices of companies that lemmy.one does not own or control, or to individuals that lemmy.one does not employ or manage.\n\n### What information do we collect?\n\n- **Basic account information:** If you register on this server, you may be asked to enter a username, an e-mail address and a password. You may also enter additional profile information such as a display name and biography, and upload a profile picture and header image. The username, display name, biography, profile picture and header image are always listed publicly.\n- **Posts, comments, and other public information:** When you submit a message, the date and time is stored as well as the application you submitted the message from. Messages may contain media attachments, such as pictures and videos. Posts are available publicly. Your posts are delivered to community subscribers, in some cases it means they are delivered to different servers and copies are stored there. When you delete or edit posts, this is likewise delivered to community subscribers. The action of crossposting or upvoting another post is always public.\n- **Direct messages:** All posts are stored and processed on the server. In some cases it means they are delivered to different servers and copies are stored there. Please keep in mind that the operators of the server and any receiving server may view such messages, and that recipients may screenshot, copy or otherwise re-share them. Do not share any sensitive information over Lemmy.\n- **IPs and other metadata:** When you log in, we record the IP address you log in from, as well as the name of your browser application. The latest IP address used is stored for up to 12 months. We also may retain server logs which include the IP address of every request to our server.\n\n### What do we use your information for?\n\nAny of the information we collect from you may be used in the following ways:\n\n- To provide the core functionality of Lemmy. You can only interact with other people's content and post your own content when you are logged in. For example, you may follow various communities to view their combined posts in your own personalized home timeline.\n- To aid moderation of the community, for example comparing your IP address with other known ones to determine ban evasion or other violations.\n- The email address you provide may be used to send you information, notifications about other people interacting with your content or sending you messages, and to respond to inquiries, and/or other requests or questions.\n\n### How do we protect your information?\n\nWe implement a variety of security measures to maintain the safety of your personal information when you enter, submit, or access your personal information. Among other things, your browser session, as well as the traffic between your applications and the API, are secured with SSL, and your password is hashed using a strong one-way algorithm.\n\n### What is our data retention policy?\n\nWe will make a good faith effort to:\n\n- Retain server logs containing the IP address of all requests to this server, in so far as such logs are kept, no more than 90 days.\n- Retain the IP addresses associated with registered users no more than 12 months.\n\nYou may irreversibly delete your account at any time.\n\n### Do we use cookies?\n\nYes. Cookies are small files that a site or its service provider transfers to your computer's hard drive through your Web browser (if you allow). These cookies enable the site to recognize your browser and, if you have a registered account, associate it with your registered account.\n\nWe use cookies to understand and save your preferences for future visits.\n\n### Do we disclose any information to outside parties?\n\nWe do not sell, trade, or otherwise transfer to outside parties your personally identifiable information. This does not include trusted third parties who assist us in operating our site, conducting our business, or servicing you, so long as those parties agree to keep this information confidential. We may also release your information when we believe release is appropriate to comply with the law, enforce our site policies, or protect ours or others rights, property, or safety.\n\nYour public content may be downloaded by other servers in the network. Your public posts are delivered to any servers which are subscribed to the community, and direct messages are delivered to the servers of the recipients, in so far as those subscribers or recipients reside on a different server than this.\n\nWhen you authorize an application to use your account, depending on the scope of permissions you approve, it may access your public profile information, your subscriptions, your messages, all your posts, your upvotes, or any other relevant account data. Refer to the privacy policy of these third-party apps for specific information about how they handle your data.\n\n### Site usage by children\n\nOur site, products and services are all directed to people who are at least 16 years old. If you are under the age of 16, per the requirements of the GDPR (General Data Protection Regulation) do not use this site.\n\nThis document is CC-BY-SA. Originally adapted from the [Discourse privacy policy](https://github.com/discourse/discourse).",
            "hide_modlog_mod_names": true,
            "application_email_admins": false,
            "actor_name_max_length": 20,
            "federation_enabled": true,
            "federation_worker_count": 65536,
            "captcha_enabled": false,
            "captcha_difficulty": "easy",
            "published": "2023-06-01T00:39:06.761646",
            "updated": "2023-07-06T17:58:31.806952",
            "registration_mode": "RequireApplication",
            "reports_email_admins": false
        }
```    

</details>
